cves/CVE-2019-11358.yml

```diff
@@ -49,7 +49,20 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The following Django vulnerability stems from the ability to use a property
+  of jQuery (a JavaScript Frontend Library) to modify internal python objects
+  which, in turn, allows for information that is not supposed to be in the
+  class object for a python class to appear or enables users to modify the
+  class object without having to directly access the code. This, at a high
+  level, is known as a cross-site scripting vulnerability, as you are putting
+  data that shouldn't exist within the application's data, in the application's
+  data.
+
+  This attack is completed by modiftying the __proto__ property of a
+  request to the backend, which in turn would allow this new object to extend
+  the object already existed. This is a major issue, as cross-site scripting
```

Comment:
change wording here or add "that" so it reads "the object that already exisited"